samples/msal-node-auth-code/index.js

```diff
@@ -4,12 +4,42 @@
  */
 const express = require("express");
 const msal = require('@azure/msal-node');
-const myLocalCache = require("./data/cache");
 const fs = require("fs");
 
 const SERVER_PORT = process.env.PORT || 3000;
 
-// initialize msal public client application
+
+const readFromStorage = () => {
+    return new Promise((resolve, reject) => {
+        fs.readFile("./data/cache.json", (err, data) => {
```

Comment:
Can these use the fs promise APIs instead?

Reply:
```js
import { promises as fs } from "fs";

const readFromStorage = () => fs.readFile("./data/cache.json");
```

Reply:
updated